fal/controllers/anime_stats.py

```diff
@@ -29,38 +29,62 @@ class AnimeStats:
 def get_forum_posts(anime: Anime) -> int:
     '''
     Requests forum posts from Jikan, then sums up all the episode discussion
-    thread replies
+    thread replies.
+
+    Where N is the weekly interval forum posts are scored:
+        if this week is not a multiple of N, return 0
+        Otherwise:
+            Gathers all the posts in the forum discussion threads of the past N weeks,
+
 
     Currently uses regex to match for Episode Discussion. This is brittle and
     will stop working once we get over 15 forum threads on an anime.
 
     ASAP we should fix Jikan's API so that it can filter for episode discussion threads
     in its search. Alternatively, we could remove the 15 search result limit but that
     seems harder to do.
+
+    Also TODO: add functionality to subtract a certain number of forum posts,
+    i.e. posts made before season started
     '''
+
     week = config.getint("weekly info", "current-week")
+    n_week = config.getint("scoring info", "forum-posts-every-n-weeks")
+
+    if week % n_week != 0:
+        return 0
+
     jikan = jikanpy.Jikan()
     forum_threads: List[Dict[str, Any]] = jikan.anime(
         anime.id, extension='forum')['topics']
 
     assert anime.name is not None
     name = anime.name  # https://github.com/python/mypy/issues/4297
 
-    episode_discussions = [thread for thread in forum_threads if re.fullmatch(
-        name + r' Episode \d{1,2} Discussion',
-        thread['title']
-    )]
+    or_alias = ""
+    if anime.alias:
+        or_alias = f"|{anime.alias}"
 
-    if len(episode_discussions) < week:
+    episode_nums_str = "|".join(
```

Comment:
Can we add a comment for this block? On first glance its a bit hard to tell that we are getting the discussions from last time we checked to now.